roles/deploy_code/tasks/cleanup.yml

```diff
@@ -6,6 +6,11 @@
   become: true
   when: "www_user != deploy_user"
 
+- name: Ensure sites/default is writable.
+  shell:
+    cmd: "if [ -d {{ deploy_path_prefix }}{{ item }} ]; then chmod 755 {{ deploy_path_prefix }}{{ item }}/{{ webroot }}/sites/default; fi"
```

Comment:
Consider replacing `/sites/default` with a `deploy_code.perms_fix_path` variable in https://github.com/codeenigma/ce-deploy/blob/1.x/roles/deploy_code/defaults/main.yml - then it won't be Drupal specific (or at least only the default value will be - we could just make it `foo` so your `if` works).

Reply:
Pushed through some changes for this!